dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -72,3 +72,7 @@ implicit.bean.scanning.disabled.CWOWB1009W.useraction=Ensure that every archive
 spi.extension.failed.to.construct.CWOWB1010E=CWOWB1010E: Failed to create CDI Extension with SPI.
 spi.extension.failed.to.construct.CWOWB1010E.explanation=CDI failed to invoke the constructor of the extension class {0} that is registered with the getExtensions() method of a CDIExtensionMetadata implementation because of the exception {1}.
 spi.extension.failed.to.construct.CWOWB1010E.useraction=Fix the error that is preventing your extension from instantiating.
+
+spi.extension.class.in.different.bundle.CWOWB1011E=CWOWB1011E: A class registered through the CDI Extension SPI was in a different bundle to the CDIExtensionMetadata implementation.
+spi.extension.class.in.different.bundle.CWOWB1011E.explanation=The class {0} was registered with the getExtensions() or getBeanClasses method of a CDIExtensionMetadata implementation is in a different bundle to the CDIExtensionMetadata implementation and will be ignored.
```

Comment:
The message is missing the `{0}` placeholder, and I don't think the explanation should have one.

also getExtensions has `()` but getBeanClasses doesn't (IIRC the message guidance says you shouldn't have brackets here)

Reply:
It would be good if we can list the names of both the class that's in the wrong archive and the `CDIMetadataExtension` implementation class which registered it.